### PR TITLE
Use a .env file

### PR DESCRIPTION
**Pull request by stunney**
Instead of having the user modify this boilerplate file, have them put their secrets in a .env file that is loaded by docker-compose be default.

### CODE REVIEW

File: docker-compose-host.yml.template [hunk 1]
```diff
@@ -1,14 +1,17 @@
-version: '2'
+version: '3.3'
 services:
   plex:
     container_name: plex
     image: plexinc/pms-docker
+    ports:
+      - "32400"
```
Comment (gbooker) [inline docker-compose-host.yml.template:7]: Ports are not specified in host networking mode and this is the wrong way to do so.

File: docker-compose-host.yml.template [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+      - PLEX_CLAIM=${CLAIM_TOKEN}
+      - PLEX_HOME=${PLEX_HOME}
     network_mode: host
     volumes:
-      - <path/to/plex/database>:/config
-      - <path/to/transcode/temp>:/transcode
-      - <path/to/media>:/data
+      - ${PLEX_HOME}/config:/config
+      - ${PLEX_HOME}/transcode:/transcode
+      - ${PLEX_HOME}/data:/data
```
Comment (gbooker) [inline docker-compose-host.yml.template:17]: Far too constraining to force all of these directories into the same directory tree and to force these specific directory names.

File: docker-compose-host.yml.template (hunk 1 above)
Comment (stunney) [reply docker-compose-host.yml.template]: I missed the host mode portion.  This was out of habbit.  Removing.

File: docker-compose-host.yml.template (hunk 2 above)
Comment (stunney) [reply docker-compose-host.yml.template]: You are correct, this should be changed.  I will break it out.

**Comment by stunney** [comment]
The CLAIM_TOKEN is 100% a secret.  The fact that it is ephemeral doesn't make it any less so.

**Pull request closed without merging**